src/MCT.js

```diff
@@ -210,6 +210,31 @@ define([
     };
 
     /**
+     * Inject build information for the About dialog et al.
+     * @private
+     */
+    MCT.prototype.specifyBuildInfo = function (buildInfo) {
+        var aliases = { timestamp: "Built" };
+        var descriptions = {
+            timestamp: "The date on which this version of Open MCT was built.",
+            revision: "A unique revision identifier for the client sources.",
+            branch: "The name of the branch that was used during the build."
+        };
+        var priorities =
+            { version: 999, timestamp: 990, revision: 995, branch: 994 };
```

Comment:
do priorities actually matter?

Reply:
For ordering, otherwise no. Happy to snip this and just take them in the order they came from `start.frag`; I'll reorder things there for consistency

Reply:
No reordering necessary after removal of priorities, will update PR with other changes momentarily